chart/cnfuzz/values.yaml

```diff
@@ -44,23 +44,33 @@ imagePullSecrets: []
 nameOverride: ""
```

Comment:
Did you verify that moving the directory does not break the expected chart uri?

Reply:
I think I checked most paths, what uri are you thinking of?